cmd/oci-image-tool/validate.go

```diff
@@ -130,20 +143,30 @@ func (v *validateCmd) validatePath(name string) error {
 
 	f, err := os.Open(name)
 	if err != nil {
-		return errors.Wrap(err, "unable to open file")
+		return "", errors.Wrap(err, "unable to open file")
 	}
 	defer f.Close()
 
+	if v.verbose {
+		fd, err := os.Open(name)
```

Comment:
@runcom , @s-urbaniak , here I've some confusion point:

for `imageLayout` and `image` type, the digests are calculated and I can collect them directly. In these 2 cases, it is meaningful to verbose output the digests which validate the consistency.

However, for `manifest`, `manifestList` and `config` type, I didn't find the digest calculation. Meantime, I think it needn't to output the digest because it just validate the simple file's json schema that is not relevant consistency.

So, my suggestion is just output digests for `imageLayout` and `image` type, and output nothing for `manifest`, `manifestList` and `config` type.

What is your proposal? THanks a lot!